common/src/main/java/org/opensearch/ml/common/conversation/ConversationalIndexConstants.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 Aryn
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.opensearch.ml.common.conversation;
+
+/**
+ * Class containing a bunch of constant defining how the conversational indices are formatted
+ */
+public class ConversationalIndexConstants {
+    /** Version of the meta index schema */
+    public final static Integer META_INDEX_SCHEMA_VERSION = 1;
+    /** Name of the conversational metadata index */
+    public final static String META_INDEX_NAME = ".conversational-meta";
```

Comment:
All plugins created by ml-commons start with `.plugins-ml` prefix. 
How about changing to `.plugins-ml-conversation-meta`